.travis.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     # Lang stage: Go
     - stage:               lang
       language:            go
-      go:                  1.10.4
+      go:                  1.13.8
```

Comment:
may be we could use `go version go1.14.2 linux/amd64`

Reply:
I was trying to use slightly older version on purpose, since go 1.14 was released just couple of months ago. I can change it though.